### ISSUE TITLE
Npc just not spawning

### ISSUE THREAD

**Issue by MrtBright** [Shark-vil/background-citizens#149]
I'm on windows 10, and It just doesn't work the option menus does show up but the npc are not found anywhere

**Comment by Shark-vil** [comment]
Have you checked the chat or received a pop-up notification after logging into the game?  Are there any warnings about missing movement mesh?

**Comment by MrtBright** [comment]
Yeah, thanks it worked it turned out i had the old version, so I downloaded
the GitHub one thanks.
